lib/node-writable.js

```diff
@@ -0,0 +1,21 @@
+var Writable = require('stream').Writable,
```

Comment:
Well, there's always [readable-stream](https://www.npmjs.com/package/readable-stream), but I'm not too concerned. I think we're fine ignoring old Nodes, as long as we make it clear what we support.